src/main/java/nl/tudelft/jpacman/board/Board.java

```diff
@@ -30,9 +30,9 @@
 	 * @return false if any square on the board is null.
 	 */
 	protected final boolean invariant() {
-		for (int x = 0; x < board.length; x++) {
-			for (int y = 0; y < board[x].length; y++) {
-				if (board[x][y] == null) {
+		for (Square[] aBoard : board) {
+			for (Square anABoard : aBoard) {
+				if (anABoard == null) {
```

Comment:
The `board[x,y]` approach doesn't seem too bad to me.

I'm also OK with the proposed alternative, but then the variable names should be better. For example

    for (Square[] row: board) {
      for (Square square: row) {
        if (square == null) ...